### PR TITLE
Add DKK-DESTR and SEK-SWESTR overnight indices

### PR DESCRIPTION
**Pull request by santorog**

### CODE REVIEW

File: modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini [hunk 1]
```diff
@@ -374,6 +374,11 @@ DKK-CITA-OIS-COMPOUND = DKK-TNR
 DKK-Tom Next = DKK-TNR
 DKK-Tom Next-OIS Compound = DKK-TNR
 
+DKK-DESTR = DKK-DESTR
```
Comment (jodastephen) [inline modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini:377]:
```suggestion
# "DKK-DESTR"/"DKK-DESTR-OIS Compound" are the ISDA 2021 codes
DKK-DESTR = DKK-DESTR
```

File: modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini [hunk 2]
```diff
@@ -444,6 +449,11 @@ SEK-STINA-COMPOUND = SEK-SIOR
 SEK-STINA-OIS-COMPOUND = SEK-SIOR
 SEK-STIBOR-OIS Compound = SEK-SIOR
 
+SEK-SWESTR = SEK-SWESTR
```
Comment (jodastephen) [inline modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini:452]:
```suggestion
# "SEK-SWESTR"/"SEK-SWESTR-OIS Compound" are the ISDA 2021 codes
SEK-SWESTR = SEK-SWESTR
```

**Review by jodastephen — CHANGES_REQUESTED** [review]

File: modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini (hunk 2 above)
Comment (santorog) [reply modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini:453]: Good point - updated like DKK DESTR using -> https://www.isda.org/a/U0WgE/2006-FRO-Mapping-to-2021-FROs-Version-5.pdf

File: modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini (hunk 1 above)
Comment (santorog) [reply modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini:378]: done

**Review by jodastephen — APPROVED** [review]

**Pull request merged**